packages/clay-css/src/scss/cadmin/components/_sidebar.scss

```diff
@@ -121,3 +145,23 @@
 		}
 	}
 }
+
+.sidebar-header-sticky {
+	position: sticky;
+	top: 0;
+	z-index: $cadmin-zindex-sticky;
+}
+
+.sidebar-footer-sticky {
+	bottom: 0;
+	position: sticky;
+	z-index: $cadmin-zindex-sticky;
+}
```

Comment:
Hey @pat270 we already have a sticky-top utility class. Why do we need the sidebar-header-sticky class? Also instead of sidebar-footer-sticky class, maybe copy the current pattern and add a sticky-bottom class instead. These would be usable in more situations.